src/Fixer/AttributeNotation/GeneralAttributeRemoveFixer.php

```diff
@@ -0,0 +1,210 @@
+    {
+        return 0;
+    }
+
+    public function isCandidate(Tokens $tokens): bool
+    {
+        return \defined('T_ATTRIBUTE') && $tokens->isTokenKindFound(T_ATTRIBUTE);
+    }
+
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
+    {
+        if (0 === \count($this->configuration['attributes'])) {
+            return;
+        }
+
+        $index = 0;
+
+        while (null !== $index = $tokens->getNextTokenOfKind($index, [[T_ATTRIBUTE]])) {
+            $attributeAnalysis = AttributeAnalyzer::collectOne($tokens, $index);
+
+            if (1 === \count($attributeAnalysis->getAttributes())) {
+                $elements = [
+                    [
+                        'name' => $attributeAnalysis->getAttributes()[0]['name'],
+                        'start' => $attributeAnalysis->getStartIndex(),
+                        'end' => $attributeAnalysis->getEndIndex(),
+                    ],
+                ];
+            } else {
+                $elements = array_map(static fn (array $attribute): array => [
+                    'name' => $attribute['name'],
+                    'start' => $attribute['start'],
+                    'end' => $attribute['end'],
+                ], $attributeAnalysis->getAttributes());
+            }
+
+            $endIndex = $attributeAnalysis->getEndIndex();
+
+            $removedCount = 0;
+            foreach ($elements as $element) {
+                $fullname = $this->determineAttributeFullyQualifiedName($tokens, $element['name'], $element['start']);
+
+                if (!\in_array($fullname, $this->configuration['attributes'], true)) {
+                    continue;
+                }
+
+                $tokens->clearRange($element['start'], $element['end']);
+                ++$removedCount;
+
+                $siblingIndex = $tokens->getNonEmptySibling($element['end'], 1);
+
+                // Clear element comma
+                if (',' === $tokens[$siblingIndex]->getContent()) {
+                    $tokens->clearAt($siblingIndex);
+                }
+            }
+
+            // Clear whole attribute if all are removed (multiline attribute case)
+            if (\count($attributeAnalysis->getAttributes()) === $removedCount) {
+                $tokens->clearRange($attributeAnalysis->getStartIndex(), $attributeAnalysis->getEndIndex());
+            }
+
+            // Clear trailing comma
+            $tokenIndex = $tokens->getMeaningfulTokenSibling($attributeAnalysis->getClosingBracketIndex(), -1);
+            if (',' === $tokens[$tokenIndex]->getContent()) {
+                $tokens->clearAt($tokenIndex);
+            }
+
+            $index = $endIndex;
+        }
+    }
+
+    protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
+    {
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('attributes', 'List of attributes to remove FQ, e.g. `Foo\Bar`.'))
```

Comment:
```suggestion
            (new FixerOptionBuilder('attributes', 'List of FQNs of attributes for removal.'))
```

the "e.g." part is not needed as examples are part of the docs.

Reply:
Fixed